plugin/restore/mysql/mysql.go

```diff
@@ -41,13 +44,13 @@ func New(driver *mysql.Driver, instance *mysqlbinlog.Instance) *Restore {
 }
 
 // RestoreBinlog restores the database using incremental backup in time range of [config.Start, config.End).
-func (r *Restore) RestoreBinlog(ctx context.Context, config mysql.BinlogInfo) error {
+func (r *Restore) RestoreBinlog(ctx context.Context, config api.BinlogInfo) error {
 	return fmt.Errorf("Unimplemented")
 }
 
 // RestorePITR is a wrapper for restore a full backup and a range of incremental backup.
```

Comment:
Yes. And also it prevents the foreign key violation in the full backup restore process.
We should refactor it later so that the first part is the current `Restore`, and the second part is just `RestoreBinlog`.

Reply:
Added TODO.